content/pages/de/jobs.rst

```diff
@@ -0,0 +1,29 @@
+Jobs
+#######
+
+:order: 2
+:date: 2021-04-02
+:icon: icon-device-laptop
+:summary: 
+:lang: de
+:slug: jobs
+
+.. container:: float-right
+
+    .. image:: {attach}/images/jobs/poetry-workshop.JPG
+        :width: 300px
+
+
+**Bist du auf der Suche nach einem Python Job?**
+
+Schau in unsere `slack`_ in #jobs-europe Kanal und #feed-jobs (auf diesem Kanal findest du mehr USA Jobs).
+
+
+
+**Hat deine Firma tolle python Jobs die du teilen möchtest?**
+
+  - Veröffentlichst du den Job direkt in unserem `slack`_ in #job-europe Kanal.
+
+  - Für Jobs außerhalb Europas füllst du dieses `Formular <http://bit.ly/pyladies-jobs-feed>`_ aus und es wird automatisch in #feed-jobs slack Kanal veröffentlicht.
```

Comment:
Für Job-Postings füllst du am besten dieses Formular aus, die Anzeige wird dann automatisch in unserem #feed-jobs Slack Kanal veröffentlicht.

Reply:
one should do that for jobs outside of Europe only.

Reply:
... automatisch im (nicht in)

Das Formular ist ausschließlich für Stellenanzeigen außerhalb Europas gedacht.